deploy/cr.yaml

```diff
@@ -419,6 +419,7 @@ spec:
 #          prefix: ""
     pitr:
       enabled: false
+#       oplogSpanMin: 10
```

Comment:
please check tabulation - line should be in the right place if you remove one symbol `#`
```suggestion
#      oplogSpanMin: 10
```